Hippo.Core/Models/OrderModels/OrderPostModel.cs

```diff
@@ -23,8 +23,8 @@ public class OrderPostModel : ProductBase
         [Range(0.0001, double.MaxValue)]
         public decimal Quantity { get; set; }
 
-        public string? InstallmentDate { get; set; }
-        public string? ExpirationDate { get; set; } //This would default to InstallmentDate + LifeCycle Months    
+        public string InstallmentDate { get; set; }
+        public string ExpirationDate { get; set; } //This would default to InstallmentDate + LifeCycle Months    
```

Comment:
@sprucely Reason for this?

Reply:
Fixes warnings. Nullable references aren't enabled for this project, so specifying those here doesn't accomplish anything.